autotest.pm

```diff
@@ -471,4 +474,25 @@ sub loadtestdir ($dir) {
     }
 }
 
+# this is called if the framework loaded a VM snapshot - all consoles
+# activated in the test's run loose their state but we should not
+# reset consoles which where already activated
+# in the last milestone.
```

Comment:
```suggestion
# This is called if the framework loaded a VM snapshot. All consoles
# activated in the test run lose their state but we should not reset
# consoles which where already activated in the last milestone.
```

(I fixed the typo "loose", made the wrapping more consistent and split it into multiple sentences. I also changed "test's run" to "test run" because I find it easier to read and means roughly the same.)